statsmodels/sandbox/stats/multicomp.py

```diff
@@ -1424,14 +1424,17 @@ def distance_st_range(mean_all, nobs_all, var_all, df=None, triu=False):
         raise ValueError('not supposed to be here')
 
     #meandiffs_ = mean_all[:,None] - mean_all
-    meandiffs = mean_all - mean_all[:,None]  #reverse sign, check with R example
-    std_pairs = np.sqrt(var_pairs)
+    meandiffs_ = mean_all - mean_all[:,None]  #reverse sign, check with R example
+    std_pairs_ = np.sqrt(var_pairs)
 
     idx1, idx2 = np.triu_indices(n_means, 1)
     if triu:
         #select all pairs from upper triangle of matrix
         meandiffs = meandiffs_[idx1, idx2]
         std_pairs = std_pairs_[idx1, idx2]
+    else:
+        meandiffs = meandiffs_
+        std_pairs = std_pairs_
```

Comment:
This is a best-guess at what is intended.  The code here is very similar to the `tukeyhsd` function.

Reply:
Looks right. Perhaps a comment 

\# These were undefined, and this appears to be the most likely intent, verify before moving out of sandbox

Reply:
It looks like this problem was introduced in ec116cce7619219ce9898732aab19ca2881e6e0a (no associated PR AFAICT) in Jan 2011.  Previously the L1430-L1434 block was commented-out.